### PR TITLE
Explanation for partials errors

### PR DESCRIPTION
**Pull request by william-gr**
Self-explanatory

### CODE REVIEW

File: src/app/explorer/farmer/farmer.component.html [hunk 1]
```diff
@@ -256,43 +256,59 @@ <h4 class="modal-title">Partial error details</h4>
...
+                  <td i18n>Signage point is invalid or was not found in pool's node blockchain. Resolution: make sure
+                    your node is synced and connected to enough peers.</td>
                 </tr>
                 <tr>
                   <td>PROOF_NOT_GOOD_ENOUGH</td>
-                  <td i18n>The proof sent is not good enough for the current difficulty of the farmer.</td>
+                  <td i18n>The proof sent is not good enough for the current difficulty of the farmer. It may happen
+                    when there is a difficulty change in the pool and there were partials already in the queue to be
+                    processed. Resolution: No action needed, it should correct itself. If the problem persists contact a
+                    pool opertator.</td>
```
Comment (atomsymbol) [inline src/app/explorer/farmer/farmer.component.html:274]: opertator ⟶ operator

Comment (william-gr) [reply src/app/explorer/farmer/farmer.component.html:274]: thanks!

File: src/app/explorer/farmer/farmer.component.html [hunk 2]
```diff
@@ -256,43 +256,59 @@ <h4 class="modal-title">Partial error details</h4>
...
-                  <td i18n>?</td>
+                  <td i18n>The partial proof is not valid. Resolution: Make sure your are running the latest version of
+                    Chia.</td>
                 </tr>
                 <tr>
                   <td>INVALID_AGG_SIGNATURE</td>
-                  <td i18n>?</td>
+                  <td i18n>The signature of the farmer does not match the farmer in the database. Resolution: Stop the
+                    farmer, remove all pool_list entries in config.yaml file and start the farmer again. If the issues
+                    persists leave and rejoin the pool.</td>
```
[the remark this answers is not in the document]
Comment (william-gr) [reply src/app/explorer/farmer/farmer.component.html:311]: I think `persists` is the proper English in this sentence.

[the remark this answers is not in the document]
Comment (atomsymbol) [reply src/app/explorer/farmer/farmer.component.html:311]: I didn't refresh the page, so I saw "issues persists".

**Pull request merged**